mysql/type.go

```diff
@@ -52,28 +52,28 @@ const TypeUnspecified = TypeDecimal
 
 // Flag information.
 const (
```

Comment:
const (
     NotNullFlag uint = 1 << iota
     PriKeyFlag
     ....
)

Reply:
`NumFlag` and `GroupFlag` have the same value, I think maybe we should hand code the values of these flags.

Reply:
ok
mysql src code reference: https://github.com/mysql/mysql-server/blob/8.0/include/mysql_com.h#L164-L166